shared/src/commonMain/kotlin/edu/usf/cutr/otp/bike_rental/api/BikeRentalApi.kt

```diff
@@ -0,0 +1,65 @@
+/*
+   Copyright University of South Florida 2021
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+
+package edu.usf.cutr.otp.bike_rental.api
```

Comment:
Still need Kotlin docs on all the API classes and public methods so apps know how to use them.